lib/ublksrv.c

```diff
@@ -8,6 +8,8 @@
 #include "ublksrv_priv.h"
 #include "ublksrv_aio.h"
 
+#define UBLK_OP_EVENTFD    0
```

Comment:
The definition conflicts with UBLK_IO_OP_READ, also please move it into include/ublksrv.h, which should be part of public API.

Same with UBLK_OP_EPOLLFD.